data/json/items/comestibles/drink.json

```
@@ -1485,49 +1485,62 @@
   {
     "id": "water",
     "type": "COMESTIBLE",
-    "comestible_type": "DRINK",
-    "category": "food",
+    "copy-from": "water_clean",
     "name": { "str_sp": "water" },
     "description": "Water, the stuff of life, necessary for all kinds of crafting, cleaning, and avoiding an agonizing death by dehydration.  You should boil or purify it before using it as drinking water.",
+    "color": "light_blue",
+    "container": "bottle_plastic",
+    "sealed": false,
+    "contamination": [ { "disease": "highly_contaminated_food", "probability": 0.1 }, { "disease": "bad_food", "probability": 1 } ]
+  },
+  {
+    "id": "water_murky",
+    "type": "COMESTIBLE",
+    "copy-from": "water",
+    "name": { "str_sp": "murky water" },
+    "description": "Water, the stuff of life.  This water looks like it has quite a bit of life in it in fact, and not the good kind.  You should filter and sterilize it if you want to drink it.",
+    "quench": 30,
+    "parasites": 5,
+    "contamination": [ { "disease": "highly_contaminated_food", "probability": 100 } ]
+  },
+  {
+    "id": "water_clean",
+    "type": "COMESTIBLE",
+    "comestible_type": "DRINK",
+    "category": "food",
     "material": [ "water" ],
     "weight": "250 g",
     "volume": "250 ml",
     "charges": 1,
     "price": 50,
     "price_postapoc": 1,
     "symbol": "~",
-    "color": "light_blue",
     "phase": "liquid",
-    "container": "bottle_plastic",
-    "sealed": false,
     "quench": 50,
     "ammo_data": { "ammo_type": "water" },
     "flags": [ "EATEN_COLD" ],
```

```suggestion
    "flags": [ "EATEN_COLD", "NUTRIENT_OVERRIDE" ],
```
I think error in `find_items_on_loops()` can be solved by adding `NUTRIENT_OVERRIDE` back to water_clean.